map_frame: world
local_frame: local
body_frame: body
imu_topic: /livox/imu
livox_topic: /livox/lidar

local_rate: 30.0
loop_rate: 1.0
publish_map_cloud: true

lio_builder:
  det_range: 100.0
  cube_len: 500.0
  resolution: 0.6
  #resolution: 0.2
  move_thresh: 1.5
  align_gravity: true
  imu_ext_rot: [1, 0, 0, 0, 1, 0, 0, 0, 1]
  imu_ext_pos: [-0.011, -0.02329, 0.04412]

localizer:
  refine_resolution: 0.1
  rough_resolution: 0.2
  refine_iter: 5
  rough_iter: 10
  thresh: 0.15
  xy_offset: 1 #0.5  1
  yaw_offset: 0 #23  0`
  yaw_resolution: 0.5 #0.034  0.5

